darts/timeseries.py

```diff
@@ -1803,6 +1803,43 @@ def plot(self,
         plt.legend()
         plt.title(self._xa.name);
 
+    def with_columns_renamed(self, cols_name: Union[List[str], str], cols_name_new: Union[List[str], str]) -> 'TimeSeries':
```

Comment:
Sorry for being annoying here, but I think `col_names` might be be better than `cols_name`.